Update README.adoc (#283)
OpenLiberty/draft-guide-gettingstarted: README.adoc

// no "try what you'll build" necessary in this guide

// Cloud hosted guide instruction
ifdef::cloud-hosted[]

> In this execrise, you need to change the user home to ***/home/project*** by the following command:

Reviewer: ```suggestion
For this guide, you need to change the user home to ***/home/project*** by running the following command:
```
